raster3d/r3.out.netcdf/r3.out.netcdf.html

```diff
@@ -260,9 +260,5 @@ <h2>SEE ALSO</h2>
 </em>
 
 <h2>AUTHORS</h2>
-S&ouml;ren Gebbert
 
-<!--
-<p>
-<i>Last changed: $Date$</i>
--->
+S&ouml;ren Gebbert
```

Comment:
`<h2>Note</h2>` should be `<h2>NOTES</h2>`, `<h2>Examples</h2> ` should be <h2>EXAMPLES</h2>.